### PR TITLE
Persisted queries docs edits

### PR DESCRIPTION
**Pull request by Meschreiber**
*Description here*

This PR duplicates relevant portions of the ["main" Persisted Queries docs](https://github.com/apollographql/docs/pull/496).

**Checklist**

Complete the checklist (and note appropriate exceptions) before a final PR is raised.

- [ ] Changes are compatible[^1]
- [x] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]. It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]. Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]. Tick whichever testing boxes are applicable. If you are adding Manual Tests:
    - please document the manual testing (extensively) in the Exceptions.
    - please raise a separate issue to automate the test and label it (or ask for it to be labeled) as `manual test`

### CODE REVIEW

File: docs/source/configuration/persisted-queries.mdx [hunk 1]
```diff
@@ -1,72 +1,164 @@
...
 
 ```yaml title="router.yaml"
 preview_persisted_queries:
   enabled: true
-  log_unknown: true
-  safelist:
-    require_id: true
-    enabled: true
+apq:
+  enabled: false # APQ must be disabled
```
Comment (Meschreiber) [inline docs/source/configuration/persisted-queries.mdx:104]: Requiring APQ be off was in the docs draft I inherited. However, I noticed this wasn't included in the config samples in https://github.com/apollographql/router/pull/3484/files.

@BrynCooke , can you confirm if users can have both apq and pq simultaneously enabled in the router?

File: docs/source/configuration/persisted-queries.mdx [hunk 2]
```diff
@@ -1,72 +1,178 @@
...
+If you want to use this security level, you should always _first_ set up [safelisting with operation strings allowed](#safelisting-operation-strings-allowed). This level requires modifying _all_ of your clients to execute operations via PQL-specified ID instead of an operation string. While making those necessary changes, you can use the less restrictive safelisting level in your router.
+
+## Router logs
+
+If a router instance is set to [audit mode](#audit-dry-run), its [logs](./logging) will include any operations it recieves that haven't been added to the PQL.
+
+For example:
+
 ```
+To-do
```
Comment (Meschreiber) [inline docs/source/configuration/persisted-queries.mdx:173]: Could you please add an example of how unregistered operations will appear in the router logs?

File: docs/source/configuration/persisted-queries.mdx [hunk 3]
```diff
@@ -1,72 +1,178 @@
...
+```
+
+If your router receives trusted operations, ....
```
Comment (Meschreiber) [inline docs/source/configuration/persisted-queries.mdx:176]: What happens for trusted operations? Do they just appear "as normal"? Could we detail that here?

File: docs/source/configuration/persisted-queries.mdx [hunk 4]
```diff
@@ -1,72 +1,178 @@
...
-* **Operation safelisting**
-  * Reject unregistered operations
-  * Require all operations to be sent as an ID
+If your organization _doesn't_ currently have an Enterprise plan, you can test this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
 
-Unlike automatic persisted queries (APQ), the ability to create a safelist of operations allows you to prevent a malicious actor from constructing a free-format query that could overload your subgraph services.
+</blockquote>
 
-***For more information on how to register queries, configure your router and update your graphql clients see the main [Persisted Query documentation](https://www.apollographql.com/docs/graphos/operations/persisted-queries).***
+With [GraphOS Enterprise](../enterprise/), you can enhance your supergraph's security by maintaining a **persisted query list** (**PQL**) for your supergraph's self-hosted router. The [Apollo Router](/router/) checks incoming requests against the PQL, a safelist of trusted operations made by your first-party apps.
```
Comment (Meschreiber) [inline docs/source/configuration/persisted-queries.mdx:14]:
```suggestion
With [GraphOS Enterprise](../enterprise/), you can enhance your supergraph's security by maintaining a **persisted query list** (**PQL**) for your supergraph's self-hosted router. The Apollo Router checks incoming requests against the PQL, a safelist of trusted operations made by your first-party apps.
```

File: docs/source/configuration/persisted-queries.mdx (hunk 1 above)
Comment (Meschreiber) [reply docs/source/configuration/persisted-queries.mdx:104]: Clarified with Adam that you must turn off APQ only for the two safelisiting modes.

**Review by chandrikas — APPROVED** [review]

**Review by BrynCooke — APPROVED** [review]
I pushed a commit to your branch with some tweaks and added a suggestion.
Otherwise this looks great and is much better than my initial draft! Feel free to merge when you are ready.

File: docs/source/configuration/persisted-queries.mdx [hunk 5]
```diff
@@ -1,72 +1,188 @@
...
+apq:
+  enabled: false # APQ must be disabled 
 ```
 
+> **Note:** To enable safelisting, you _must_ disable [automatic persisted queries](./in-memory-caching#caching-automatic-persisted-queries-apq).
+
+If you want to use this security level, you should always _first_ set up [safelisting with operation strings allowed](#safelisting-operation-strings-allowed). This level requires modifying _all_ of your clients to execute operations via PQL-specified ID instead of an operation string. While making those necessary changes, you can use the less restrictive safelisting level in your router.
+
+
+## Router logs
```
[the remark this answers is not in the document]
Comment (Meschreiber) [reply docs/source/configuration/persisted-queries.mdx]: Good point!

[the remark this answers is not in the document]
Comment (Meschreiber) [reply docs/source/configuration/persisted-queries.mdx]: https://github.com/apollographql/router/pull/3501/commits/4d25d6ebf7540f41363a6c478ecd82e33614ad04

**Pull request merged**